app/serializers/v1/user_serializer.rb

```diff
@@ -3,25 +3,44 @@ class UserSerializer < ApplicationSerializer
     attributes :email
 
     attribute :first_name do |user|
-      unless user.person.nil?
-        user.person.first_name
+      if person = user.person
+        person.first_name
       end
     end
 
     attribute :last_name do |user|
-      unless user.person.nil?
-        user.person.last_name
+      if person = user.person
+        person.last_name
       end
     end
 
     attribute :image_url do |user|
-      unless user.person.nil?
-        user.person.image_url
+      if person = user.person
+        person.image_url
+      end
+    end
+
+    attribute :hub do |user|
+      if person = user.person
+        person&.hub&.name
       end
     end
 
     belongs_to :person, serializer: V1::PersonSerializer, id_method_name: :external_identifier do |user|
       user.person
     end
+
+    attribute :ssj do |user|
+      if person = user.person
+        ssj_team = person.ssj_team
+        workflow = ssj_team.workflow
+        {
+          currentPhase: workflow.current_phase,
+          opsGuide: V1::PersonSerializer.new(ssj_team.ops_guide),
+          regionalGrowthLead: V1::PersonSerializer.new(ssj_team.regional_growth_lead),
+          expectedStartDate: ssj_team.expected_start_date
+        }
+      end
+    end
```

Comment:
the SSJ stuff i might have put on the ssj dashboard serializer.  like there should be 1 serializer that has all the data to render that SSJ dashboard page.  eventually there will be many other apps for users and we'll keep loading this ssj data.  not an issue now but this definitely belongs on the ssj dashboard.